README.md

```diff
@@ -100,20 +130,20 @@ docker compose build
 docker compose run --rm util_caching
 ```
 
-The library is installed in the Docker image under `/usr/local/include/util_caching/` and `/usr/local/lib/cmake/util_caching/`.
+The library is installed in the Docker image under `/usr/include/util_caching/` and `/usr/lib/cmake/util_caching/`.
```

Comment:
Oh, I see. I thought it was just outdated because I have the .deb version installed locally. I don't think forcing another path is necessary. I corrected the path in [1988385](https://github.com/KIT-MRT/util_caching/pull/41/commits/1988385b30505478c9ca57ad5ad9f1116bc05bc3).